hpx/runtime/threads/cpu_mask.hpp

```diff
@@ -159,6 +159,10 @@ namespace hpx { namespace threads
         mask.set(idx);
     }
 
+    inline void unset(mask_type& mask, std::size_t idx)
+    {
+	mask.set(idx,0);
+    }
```

Comment:
Please follow the formatting of the surrounding code. Alternatively use clang-format (with the `.clang-format` file in the root of this project) to format the code sections you touched.